streams/src/main/java/org/apache/kafka/streams/StoreQueryParams.java

```diff
@@ -21,12 +21,11 @@
 import java.util.Objects;
 
 /**
- * Represents all the query options that a user can provide to state what kind of stores it is expecting.
+ * Represents all the query options that a user can provide to state what kind of stores we are expecting.
  * The options would be whether a user would want to enable/disable stale stores
```

Comment:
Cosmetic: ` Options could indicate - whether we want stale stores to be included (or) we want to filter for only stores belonging to a partition`.

Reply:
Done.